tests/fuzzing/rw_fuzzer.py

```diff
@@ -0,0 +1,24 @@
+import sys
+
+import atheris
+from fuzz_helpers import EnhancedFuzzedDataProvider
+
+with atheris.instrument_imports(include=["jupytext"]):
+    import jupytext
+
+
+def TestOneInput(data: bytes):
+    fdp = EnhancedFuzzedDataProvider(data)
+
+    py = fdp.ConsumeRemainingString()
+    nb = jupytext.reads(py, "py")
+    jupytext.writes(nb, "py")
```

Comment:
I would like to understand what is the point of this fuzzer compared to the other one?